spec/models/transac_spec.rb

```diff
@@ -0,0 +1,27 @@
+require 'rails_helper'
+
```

Comment:
- [OPTIONAL] Excellent adding unit tests for all the most important components of the RoR application 👍🏻 Let's now add more integration tests with capybara. 👍🏻